### PR TITLE
1.19.4

### PR DESCRIPTION
**Pull request by vadimpronin**
The data was extracted using Burger (recipes) and minecraft-data-generator-server (everything else).
The protocol.json wasn't updated as https://wiki.vg/Protocol doesn't show any changes (yet?)

### CODE REVIEW

**Comment by extremeheat** [comment]
Please also update the protocolVersion.json file, it seems the automated bot didn't update it

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Language independent module providing minecraft data for minecraft clients, serv
 
 Supports
 * Minecraft PC version 0.30c (classic), 1.7.10, 1.8.8, 1.9 (15w40b, 1.9, 1.9.1-pre2, 1.9.2, 1.9.4),
```
Comment (AugustinMauroy) [inline README.md:11]:
```suggestion
* Minecraft  Java Edition, 1.7.10, 1.8.8, 1.9 (15w40b, 1.9, 1.9.1-pre2, 1.9.2, 1.9.4),
```

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Language independent module providing minecraft data for minecraft clients, serv
...
- 1.10 (16w20a, 1.10-pre1, 1.10, 1.10.1, 1.10.2), 1.11 (16w35a, 1.11, 1.11.2), 1.12 (17w15a, 17w18b, 1.12-pre4, 1.12, 1.12.1, 1.12.2), 1.13 (17w50a, 1.13, 1.13.1, 1.13.2-pre1, 1.13.2-pre2, 1.13.2), 1.14 (1.14, 1.14.1, 1.14.3, 1.14.4), 1.15 (1.15, 1.15.1, 1.15.2), 1.16 (20w13b, 20w14a, 1.16-rc1, 1.16, 1.16.1, 1.16.2, 1.16.3, 1.16.4, 1.16.5), 1.17, 1.17.1, 1.18 (1.18, 1.18.1, 1.18.2), 1.19 (1.19, 1.19.2, 1.19.3)
+ 1.10 (16w20a, 1.10-pre1, 1.10, 1.10.1, 1.10.2), 1.11 (16w35a, 1.11, 1.11.2), 1.12 (17w15a, 17w18b, 1.12-pre4, 1.12, 1.12.1, 1.12.2), 1.13 (17w50a, 1.13, 1.13.1, 1.13.2-pre1, 1.13.2-pre2, 1.13.2), 1.14 (1.14, 1.14.1, 1.14.3, 1.14.4), 1.15 (1.15, 1.15.1, 1.15.2), 1.16 (20w13b, 20w14a, 1.16-rc1, 1.16, 1.16.1, 1.16.2, 1.16.3, 1.16.4, 1.16.5), 1.17, 1.17.1, 1.18 (1.18, 1.18.1, 1.18.2), 1.19 (1.19, 1.19.2, 1.19.3, 1.19.4)
 * Minecraft bedrock version 0.14, 0.15, 1.0, 1.16.201, 1.16.210, 1.16.220, 1.17.0, 1.17.10, 1.17.30, 1.17.40, 1.18.0, 1.18.11, 1.18.30, 1.19.1, 1.19.10, 1.19.20, 1.19.21, 1.19.30, 1.19.40, 1.19.50, 1.19.60, 1.19.62, 1.19.63, 1.19.70
```
Comment (AugustinMauroy) [inline README.md:13]:
```suggestion
* Minecraft Bedrock Edition 0.14, 0.15, 1.0, 1.16.201, 1.16.210, 1.16.220, 1.17.0, 1.17.10, 1.17.30, 1.17.40, 1.18.0, 1.18.11, 1.18.30, 1.19.1, 1.19.10, 1.19.20, 1.19.21, 1.19.30, 1.19.40, 1.19.50, 1.19.60, 1.19.62, 1.19.63, 1.19.70
```

**Comment by rom1504** [comment]
anyone wants to help review this ?

eg
* check diffs with 1.19.3 for a few files (blocks.json items.json ..) and make sure it's reasonable

**Review by AugustinMauroy — APPROVED** [review]
LTGM

**Comment by rom1504** [comment]
@AugustinMauroy how did you review it ?

**Comment by AugustinMauroy** [comment]
@rom1504 I had compare to 1.19.3

**Comment by rom1504** [comment]
So please send that comparison then

**Comment by faaxz** [comment]
will this ever be merged?

**Comment by rom1504** [comment]
If someone reviews it yes

**Review by andriycraft — APPROVED** [review]

**Comment by NotSugden** [comment]
do `data/pc/common/protocolVersions.json` & `data/pc/common/versions.json` not need to also be updated to add version 1.19.4

**Comment by 3jq** [comment]
> still nobody wanting to perform basic sanity check on this ?

Okay, so just tell me what do you need from a review and I will.

**Comment by 3jq** [comment]
> > still nobody wanting to perform basic sanity check on this ?
> 
> Okay, so just tell me what do you need from a review and I will.

Except what you've said already

**Review by 3jq — APPROVED** [review]
Compared 1.19.3 data and 1.19.4 data, used minecraft 1.19.4 changelog (https://www.minecraft.net/en-us/article/minecraft-java-edition-1-19-4)

Everything looks fine to me.
He also did some changes in README (about formatting), also changed PC version to Java Edition, which is kinda right.

However, I'm not sure about the protocol still - #703 has it.
Also https://wiki.vg/Protocol is saying that their protocol is up-to-date.
I asked another bot developer that updated mc-data by himself for his protocol, hopefully he's gonna share it.

File: README.md (hunk 1 above)
Comment (rom1504) [reply README.md:11]: No

**Comment by rom1504** [comment]
https://github.com/PrismarineJS/minecraft-data/blob/master/.github/ISSUE_TEMPLATE/new_version.md

**Pull request merged**